docs/Hosts/Spawn-Hosts.md

```diff
@@ -173,6 +166,21 @@ This can also be done through the CLI:
 evergreen host stop --host <host_id> --keep-off
 ```
 
+#### FAQ
+
+**Q: Can I still manually stop and start my hosts if the schedule is running?**
+A: Yes! Even though Evergreen will automatically stop and start your host on the configured schedule, you can still
+manually stop and start your host whenever you need to. If you do this, it's recommended, you also [set a temporary
+exemption](#temporary-exemptions) to ensure that you can use your host without being accidentally interrupted when the
+sleep schedule tries to automatically stop/start your host.
+
+**Q: Is there a way to get a hostname that doesn't change when the host restarts?**
```

Comment:
I think that the "Is there a way to" wording might read as "It is configurable". 
Can we change it to something like: 
`Will the hostname be persistent through restarts?`
or
`Will the hostname change every time the host restarts?`

Reply:
Sure, I updated it to the first one (kind of).